accesses/services/accesses.py

```diff
@@ -487,4 +495,161 @@ def close_accesses(perimeters_to_delete: QuerySet):
         _logger.info(f"{len(accesses_to_delete)} accesses have been closed: {accesses_to_delete}")
 
 
-accesses_service = AccessesService()
\ No newline at end of file
+accesses_service = AccessesService()
+
+
+@dataclass
+class FhirPractitioner:
+    username: str
+    firstname: str
+    lastname: str
+
+
+@dataclass
+class FhirPractitionerRole:
+    id: str
+    active: bool
+    practitioner_id: str
+    perimeter_id: int
+    role_name: str
+    start_datetime: datetime
+    end_datetime: datetime
+
+
+class AccessesSynchronizer:
+
+    def __init__(self):
+        token = jwt_auth_service.generate_system_token()
+        self.fhir_client = SyncFHIRClient(url=AccessConfig.FHIR_URL,
+                                          authorization=f"Bearer {token}")
+
+    @staticmethod
+    def log(message: str):
+        _logger.info(f"[ORBIS sync] {message}")
+
+    def sync_orbis_resources(self):
+        self.sync_profiles()
```

Comment:
comment this (this will not be used), we just want to sync the accesses for existing profiles/users for now